### ISSUE TITLE
Any examples of this working in a k8s environment?

### ISSUE THREAD

**Issue by eliasdarruda** [beardedeagle/mnesiac#93]
## Question

I'm wondering how would it work in a k8s environment with a multi node or single node volume.
Does anyone have a working example already?

**Comment by beardedeagle** [comment]
Yes, mnesiac should work in kubernetes though I don't have an example readily available, apologies. Might be good to get something added to the docs about it at some point though.

**Comment by eliasdarruda** [comment]
I've tried to use disc_only copies with multiple pods, the problem is that each pod has its own unique hostname, and it ends up creating ghost mnesia copies that ends up blocking other new pods rolling out

I'll try to use a statefulset to keep the hostname predictable across multiple pods.

I'll keep adding here what I can find for those who want to use disc only copies with k8s

**Comment by acrolink** [comment]
Any news on this? Working examples?

**Comment by eliasdarruda** [comment]
I've managed to write an example here:

https://github.com/eliasdarruda/mnesia-with-k8s

This is a working version using Mnesiac and kubernetes with StatefulSet, this is saving and recovering the state correctly on rollouts and new setups.

@acrolink

**Comment by acrolink** [comment]
@eliasdarruda Thank you very much for sharing.
